lib/command/maintenance_set_page.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Command
+  class MaintenanceSetPage < Base
+    NAME = "maintenance:set-page"
+    USAGE = "maintenance:set-page PAGE_URL"
+    REQUIRES_ARGS = true
+    OPTIONS = [
+      app_option(required: true)
+    ].freeze
+    DESCRIPTION = "Sets the page for maintenance mode"
+    LONG_DESCRIPTION = <<~DESC
+      - Sets the page for maintenance mode
+      - Only works if the maintenance workload uses the `shakacode/maintenance-mode` image
+      - Will set the URL as an env var `PAGE_URL` on the maintenance workload
+      - Specify the maintenance workload through `maintenance_workload` in the `.controlplane/controlplane.yml` file
```

Comment:
Does this name really need to be configurable?

Why not always use `maintenance-workload`